apps/backend/lib/src/pagination.ts

```diff
@@ -0,0 +1,13 @@
+import { Request, Response, NextFunction } from "express";
+// Pagination middleware
+export const usePagination = (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  const num_items = parseInt(req.query.num_items as string) || 10;
+  const offset = parseInt(req.query.offset as string) || 0;
+  req.body.num_items = num_items;
+  req.body.offset = offset;
+  next();
```

Comment:
Do we really need this as a middleware? On a per route basis seems better since its then apparently obvious any developer what these magic values are (locality of behaviour)